tests/rptest/clients/rpk_remote.py

```diff
@@ -53,6 +53,13 @@ def tune(self, tuner):
     def mode_set(self, mode):
         return self._execute([self._rpk_binary(), 'redpanda', 'mode', mode])
 
+    def redpanda_start(self, additional_args="", env_vars="", log_file=""):
+        return self._execute([
+            env_vars,
+            self._rpk_binary(), "redpanda", "start", "-v", additional_args,
+            ">>", log_file, "2>&1", "&"
```

Comment:
Does ">> 2&1" (i.e. the `>>` but no filename) actually work if the default "" is set for log_file?

Reply:
You are right, I'll make log_file required.